SonarQube.Common/ProcessRunner.cs

```diff
@@ -64,6 +87,16 @@ public bool Execute(string exeName, string args, string workingDirectory, int ti
                 WorkingDirectory = workingDirectory
             };
 
+            if (envVariables != null)
+            {
+                foreach (KeyValuePair<string, string> envVariable in envVariables)
+                {
+                    if (!String.IsNullOrEmpty(envVariable.Value))
+                    {
+                        psi.EnvironmentVariables.Add(envVariable.Key, envVariable.Value);
```

Comment:
Child processes do not automatically inherit the process env variables of the parent but have to be manually added